broker/service/api/v10.py

```diff
@@ -27,12 +29,20 @@
 
 API_LOG = Log("APIv10", "logs/APIv10.log")
 
-submissions = {}
 clusters = {}
 activated_cluster = None
 
+
 CLUSTER_CONF_PATH = "./data/clusters"
 
+if api.persistence_name == 'etcd':
```

Comment:
is this going to break if I don't add [persistence] section to my config? `api` won't have such an attribute.

Reply:
this whole block needs to be refactored to consider when the user does not want a persistence backend. latter on this code we rely on a `persistence_obj` object that would not exist in such a scenario.